tests/conftest.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+This is the initial module that contains the pytest configuration fixtures
+"""
+import pytest
+import os
+from app import create_app
+from app import db as _db
+from sqlalchemy import event
+from sqlalchemy.orm import sessionmaker
+from config import TestingConfig
+
+
+@pytest.fixture(scope='session')
+def app(request):
+    """
+    This is creates the mock app for testing, it uses the
+    TestingConfig in Config.py
+    """
+    app = create_app(config_settings=TestingConfig)
+    ctx = app.test_request_context()
```

Comment:
Added the suggestion, after reading it is actually the right thing to do.  @liamocn give it another try.